components/ai_chat/resources/page/api/index.ts

```diff
@@ -53,6 +55,11 @@ class PageAPI extends API<State> {
   public conversationEntriesFrameObserver: Mojom.ParentUIFrameCallbackRouter
     = new Mojom.ParentUIFrameCallbackRouter()
 
+  public tabInformer: Mojom.TabInformerRemote
+    = Mojom.TabInformer.getRemote()
+
+  private tabListenerReceiver = new Mojom.TabListenerReceiver(this)
```

Comment:
I don't mind using the interface pattern instead of the CallbackRouter pattern, but maybe we should decide when instead of an arbitrary mix?

Reply:
Updated to use the `CallbackRouter` for consistency